_saas-integrations/google-sheets/v1/google-sheets-v1.md

```diff
@@ -118,7 +122,7 @@ setup-steps:
 #     Replication Details    #
 # -------------------------- #
 
-replication sections:  
+replication-sections:  
```

Comment:
I fixed this - this was why none of the info in this section was showing up. It needs to be `replication-sections`, and if you look at the diff (the left side of the *Files Changed* tab), you'll see it was `replication sections` before.

The template this page uses will always look for `replication-sections`